CHANGELOG.md

```diff
@@ -3,6 +3,14 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [vector-0.37.0] - 2024-12-04
```

Comment:
I usually generate the changelog in a subsequent PR. Example from 0.42.0 release:

- First PR bumping the Vector version: https://github.com/vectordotdev/helm-charts/pull/420. This didn't need to bump the chart version because it had already been bumped (but not yet released)
- Second PR generating the changelog: https://github.com/vectordotdev/helm-charts/pull/421

The reason for splitting up the PRs is that `git-cliff` generates better changelogs (the commit hash points to a released commit rather than a commit on a PR branch).